ci-operator/config/openshift/cert-manager-operator/openshift-cert-manager-operator-cert-manager-1.17.yaml

```diff
@@ -84,12 +84,18 @@ tests:
     test:
     - ref: fips-check-image-scan
 - as: e2e-operator
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    owner: openshift-ci
+    product: ocp
+    timeout: 2h0m0s
+    version: "4.19"
+  skip_if_only_changed: ^(docs/|deploy/|jsonnet/)|\.md$|^(?:.*/)?(?:\.gitignore|OWNERS|PROJECT|LICENSE)$
   steps:
-    cluster_profile: aws-2
     dependencies:
       OO_BUNDLE: cert-manager-operator-bundle
     env:
-      FIPS_ENABLED: "true"
```

Comment:
This is a tradeoff to adapt generic-claim. Given that in QE CI there are good mount of FIPS env coverage, hence to remove it safely